.github/workflows/build.yml

```diff
@@ -11,6 +11,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: save check run id
+        run: |
+          CHECK_SUITE_URL=$(curl -s -H "authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID} | jq -r '.check_suite_url')
+          CHECK_RUN_ID=$(curl -s -H "authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" -H "Accept: application/vnd.github.antiope-preview+json" $CHECK_SUITE_URL/check-runs | jq '.check_runs[] | select(.name=="build-and-verify") | .id ')
+          mkdir -p ./pr
+          echo $CHECK_RUN_ID > ./pr/run-id
```

Comment:
can we call this file `build-verify-check-run-id` to reflect what is the value is?

Reply:
Done. 6bb4c0d8fd39a892f391f3c7eead832a1985f031